etl/steps/data/meadow/papers/2022-11-04/riley_2005/riley_2005.data.csv

```diff
@@ -0,0 +1,56 @@
+year,entity,life_expectancy
+1770,Europe,34.3
+1800,Europe,33.3
+1820,Europe,35.6
+1850,Europe,36.3
+1870,Europe,36.2
+1900,Europe,42.7
+1913,Europe,46.8
+1950,Europe,64.7
+1973,Europe,70.9
+1990,Europe,74.2
+2001,Europe,76.8
+1885,Asia,27.5
+1900,Asia,28
+1913,Asia,28.1
+1950,Asia,41.6
+1973,Asia,57.5
+1990,Asia,64.5
+2001,Asia,67.1
+1920,Africa,26.4

```

Comment:
Shouldn't this be 1925? Given that you are taking the average year in all other intervals, if I understand correctly.

Reply:
That is totally correct, thanks pablo!